stock_analysis/reports/stock_analysis.py

```diff
@@ -0,0 +1,48 @@
+# -*- coding: utf-8 -*-
+# © 2016 Lorenzo Battistini - Agile Business Group - I.A.S. Ingenieria,
+# Aplicaciones y Software
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import models, fields, tools
+
+
+class StockAnalysis(models.Model):
+    _name = 'stock.analysis'
+    _auto = False
+    _rec_name = 'product_id'
+
+    product_id = fields.Many2one(
+        'product.product', string='Product', readonly=True)
+    location_id = fields.Many2one(
+        'stock.location', string='Location', readonly=True)
+    qty = fields.Float(string='Quantity', readonly=True)
+    lot_id = fields.Many2one(
+        'stock.production.lot', string='Lot', readonly=True)
+    package_id = fields.Many2one(
+        'stock.quant.package', string='Package', readonly=True)
+    in_date = fields.Datetime('Incoming Date', readonly=True)
+    categ_id = fields.Many2one(
+        'product.category', string='Category', readonly=True)
+    company_id = fields.Many2one(
+        'res.company', string='Company', readonly=True)
+
+    def init(self, cr):
+        tools.drop_view_if_exists(cr, 'stock_analysis')
+        cr.execute(
+            """CREATE or REPLACE VIEW stock_analysis as (
```

Comment:
Again, split this query in subparts. The errors in Travis are because you are not doing correctly the split, not because the split is incorrect.